language-server/test/dotty/tools/languageserver/util/server/TestServer.scala

```diff
@@ -15,15 +15,18 @@ class TestServer(testFolder: Path) {
 
   private[this] def init(): InitializeResult = {
     // Fill the configuration with values populated by sbt
-    def showSeq[T](lst: Seq[T]): String = lst.map(elem => '"' + elem.toString + '"').mkString("[ ", ", ", " ]")
+    def showSeq[T](lst: Seq[T]): String =
+      lst
+        .map(elem => '"' + elem.toString.replace('\\', '/') + '"')
+        .mkString("[ ", ", ", " ]")
     val dottyIdeJson: String =
       s"""[ {
          |  "id" : "dotty-ide-test",
          |  "compilerVersion" : "${BuildInfo.ideTestsCompilerVersion}",
          |  "compilerArguments" : ${showSeq(BuildInfo.ideTestsCompilerArguments)},
          |  "sourceDirectories" : ${showSeq(BuildInfo.ideTestsSourceDirectories)},
          |  "dependencyClasspath" : ${showSeq(BuildInfo.ideTestsDependencyClasspath)},
-         |  "classDirectory" : "${BuildInfo.ideTestsClassDirectory}"
+         |  "classDirectory" : "${BuildInfo.ideTestsClassDirectory.toString().replace('\\','/')}"
```

Comment:
No need for the parentheses after `toString`

Reply:
Sorry, Java habits.  Will fix.
Should I amend or just new commit is OK?